draft-birkholz-scitt-scrapi.md

```diff
@@ -217,6 +244,38 @@ Security Considerations
 
 Maybe
 
+## Media Type Registration
+
+This section requests registration of the "application/receipt+cose" media type {{RFC2046}} in
+the "Media Types" registry in the manner described in {{RFC6838}}.
+
+TODO: Consider negotiation for receipt as "JSON" or "YAML".
```

Comment:
What's the payoff for having a YAML version?

Reply:
YAML is growing in popularity, for cases where both humans and machines process the file, due to its readability and writability.

You could argue its a bad thing, if people start reading yaml instead of just using a machine to check it.

The point of the comment was to address requesting a representation of a receipt that a browser or filesystem might display nicely.

Reply:
OK got you. It's more easily transmissible in a web/REST world and more supported by standard tooling than pure COSE/CBOR. Makes a lot of sense, our implementation wraps in JSON for the same reasons.